.github/workflows/dbt_slim_ci.yml

```diff
@@ -87,8 +87,13 @@ jobs:
           [[ -z "$test" ]] && { echo "Success"; exit 0; } || { echo "Found active child models of removed models:"; echo "$test"; exit 1; }
 
       - name: dbt seed
+        if: ${{ !contains(github.event.pull_request.body,'CI:refresh-all-seeds') }}
```

Comment:
if you're adding a whole new step, why is this conditional needed?

Reply:
Basically we want to do:
```
if (PR contains "CI:refresh-all-seeds"):
  dbt seed @state:modified
else:
  dbt seed state:modified
  ```
 But since GitHub workflows don't have an if-else pattern, we need 2 steps with inverted conditionals.

Reply:
ah, if doesn't contain, got it